frontend/packages/dev-console/integration-tests/support/step-definitions/addFlow/create-from-container-image.ts

```diff
@@ -72,6 +72,10 @@ When('user selects the {string} from Runtime Icon dropdown', (runTimeIcon: strin
   containerImagePage.selectRunTimeIcon(runTimeIcon);
 });
 
+When('user enters Name as {string}', (name: string) => {
```

Comment:
@Lucifergene is this used anywhere?

Reply:
@divyanshiGupta Have removed the changes as the e2e tests are working fine now.

/cc @sanketpathak